src/Exclusion/Check/ExcludeByDocBlock.php

```diff
@@ -0,0 +1,36 @@
+<?php
+declare(strict_types=1);
+
+namespace Rector\Exclusion\Check;
+
+use PhpParser\Comment\Doc;
+use PhpParser\Node;
+use PhpParser\Node\Stmt;
+use Rector\Contract\Exclusion\ExclusionCheckInterface;
+use Rector\Contract\Rector\PhpRectorInterface;
+use Rector\NodeTypeResolver\Node\AttributeKey;
+
+final class ExcludeByDocBlock implements ExclusionCheckInterface
```

Comment:
The class name seems like value object.
It's convention to respect the interface name here ↓

`ExcludeByDocBlockExclusionCheck`

Also better naming as role to prevent confusion with value object:

- value vs role
- Check vs Checker
- object vs service

Reply:
I renamed it to ExcludeByDocBlockExclusionCheck as suggested

Reply:
Thanks :+1: